### ISSUE TITLE
add patchouli support[suggestion]

### ISSUE THREAD

**Issue by rtb523** [DaedalusGame/EmbersRekindled#287]
there is a guidebook mod called patchouli and you should add one

**Comment by rtb523** [comment]
also it allows a blue print of multiblocks

**Comment by DaedalusGame** [comment]
except that uhhh embers multiblocks don't need blueprints because they just place the full multiblock

**Comment by rtb523** [comment]
yeah but thats not the only thing it does it also is a guidebook like a questbook or somthing with out quests

**Comment by rtb523** [comment]
just the blueprint part is a feuture

**Comment by rtb523** [comment]
you can look at there page for a better explanation
